TUTORIAL.rst

```diff
@@ -0,0 +1,187 @@
+          at 10K and the last (44) at 300K. The samples increase in
+          temperature as their numbers increase. The "C" in their names
+          indicates that they have undergone cooling.
+
+        * Note that these files have the ``.gr`` extension, which
+          indicates that they are measured PDFs. The ``.cgr`` file
+          extension indicates that a file is a calculated PDF, sich as
+          those generated by the `PDFgui <https://www.diffpy.org/products/pdfgui.html>`_
+          program.
+
+    4. First, we will run the PDFmorph application without any morphing
+       and only using one PDF. Type the following command into your
+       Command Line ::
+
+            pdfmorph darkSub_rh20_C_01.gr darkSub_rh20_C_01.gr
+
+        This should produce two PDF curves which are congruent, resulting
+        in a flat green line underneath them.
+
+    5. Now, we will see PDFmorph run with two different PDFs and no
+       morphing. Type the following command into your command line ::
+
+            pdfmorph darkSub_rh20_C_01.gr darkSub_rh20_C_44.gr
+
+       Without morphing, the difference Rw = 0.407. This indicates that
+       the two PDFs vary drastically.
+
+            * While running the ``pdfmorph`` command, it is important
+              to remember that the first PDF file argument you provide
+              (in this case, ``darkSub_rh20_C_01.gr``) is the PDF which
+              will get morphed, while the second PDF file argument you
+              provide (here, ``darkSub_rh20_C_44.gr``) is the PDF which
+              acts as the model and does not get morphed. Hereinafter,
+              we will refer to the first PDF argument as the "objective"
+              and the second as the "reference", as the PDFmorph display
+              does.
+
+    6. Now, we will start the morphing process, which requires us to
+       provide initial guesses for our scaling factor, Gaussian smear,
+       and stretch, separately. We will start with the scaling factor.
+       Begin by typing the command ::
+
+            pdfmorph --scale=2 -a darkSub_rh20_C_01.gr darkSub_rh20_C_44.gr
+
+       Now, the difference Rw = 1.457, a significant increase from our
+       value previously. We must modify our initial value for the
+       scaling factor and do so until we see a reduction in the
+       difference Rw from the unmorphed value. Type ::
+
+            pdfmorph --scale=0.9 -a darkSub_rh20_C_01.gr darkSub_rh20_C_44.gr
+
+       The difference Rw is now 0.351, lower than our unmorphed
+       example's value. To see PDFmorph optimize the scale factor,
+       simply drop ``-a`` from the command and type ::
+
+            pdfmorph --scale=0.9 darkSub_rh20_C_01.gr darkSub_rh20_C_44.gr
+
+       PDFmorph, given a reasonable initial guess, will use find the
+       optimal value for each morphing feature. Here, we see that
+       PDFmorph displays ``scale = 0.799025`` in the command prompt,
```

Comment:
I am pretty sure that PDFmorph will successfully converge with the default initial guess, or even with the bad initial guess of 2 (not sure about that....did you test it?) so this may not be a great example of introducing the concept of the initial guess.  The default use of the program should be that you don't give it initial guesses unless it doesn't converge.